test/integration/smart_answer_flows/pip_checker_test.rb

```diff
@@ -55,29 +55,29 @@ class PIPCheckerTest < ActiveSupport::TestCase
   context "getting DLA" do
     setup do
       add_response 'yes'
-      Timecop.travel('2014-02-03')
+      Timecop.travel('2013-02-03')
```

Comment:
No worries @ikennaokpala. It [seems to be reset here for all tests](https://github.com/alphagov/smart-answers/blob/e275244a0464576805108edd337fdf665a802ffb/test/test_helper.rb#L26-L26).